src/components/common/app-title-cols.vue

```diff
@@ -1,13 +1,13 @@
 <template>
-  <div>
-    <v-col cols="12" class="mb-4">
-      <Stats v-if="!isMobile" class="pr-10"/>
-    </v-col>
-    <v-col cols="12" class="mb-n2">
+  <v-row no-gutters>
+    <v-col cols="12" md="6" lg="6" class="mt-4" :class="[{'pt-10': !isMobile}, 'pl-2']">
```

Comment:
Might as well just move "pl-2" to class property.

Reply:
For sure, will do.

Reply:
Oops, missed this. Updated 👍